backend/app/literature/crud/resource_crud.py

```diff
@@ -118,20 +118,20 @@ def destroy(db: Session, curie: str):
     return None
 
 
-def patch(db: Session, curie: str, resource_update: Union[ResourceSchemaUpdate, dict]) -> dict:
+def patch(db: Session, curie: str, resource_update: ResourceSchemaUpdate) -> dict:
```

Comment:
I think that Ian added the Union here when he was making the tests. I significantly prefer what you are changing it to. That being said you should make sure the tests still pass!

Reply:
Yes I added it. Patch fails if you pass it a ResourceSchemaUpdate but fine with a dict. This goes for most of the patch methods in all the schema files. See tests for examples.